rest-api/src/main/java/gov/cms/qpp/conversion/api/services/DbServiceImpl.java

```diff
@@ -99,7 +102,7 @@
 	 */
 	public Metadata getMetadataById(String uuid) {
 		API_LOG.info("Read item {} from DynamoDB", uuid);
-		return mapper.load(Metadata.class, uuid);
+		return mapper.get().load(Metadata.class, uuid);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "mapper.isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3655)